draft-ietf-trans-rfc6962-bis.md

```diff
@@ -681,6 +678,30 @@ value will not be a valid X.509v3 signature that could be used in conjunction
 with the TBSCertificate (from `SignedData.encapContentInfo.eContent`) to
 construct a valid certificate.
 
+### Binding Intent to Issue  {#binding_intent_to_issue}
+
+Under normal circumstances, there will be a short time period between
+precertificate submission and issuance of the corresponding certificate.
+Occasional delays are to be expected though (e.g., due to log server downtime),
```

Comment:
Nit "Longer delays" ? It seems to me this case is distinct from the "short time period between precertificate submission and issuance of the corresponding certificate" so may be worth referring to it directly.

Reply:
Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/1a231c5a5300af1739842551fe373ec63ad4cd36.